pom.xml

```diff
@@ -618,5 +618,60 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>format</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
```

Comment:
Are we sure we want reformatting to be `activeByDefault`?
Or should it be that `maven-checkstyle-plugin` is `activeByDefault`?

Reply:
if we run formatting by default, there is not need to run checkstyle. It's just optional to make it available, but at the moment, the result of reformatting does not remove any of the checkstyle violations.

Reply:
Does checkstyle report violations even after formatter runs and adjusts everything though?

Reply:
Ok, I see now that checkstyle logs a huge number of warnings and the build fails when it is activated, so I see now why you didn't activate it by default.